DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/traffic/ixnetwork/ixnetwork_ixia_client_impl.py

```diff
@@ -110,22 +111,34 @@ def run_connect(self, device, command, *argv, **kwarg):
                 pports.append({'Arg1': pport[0], 'Arg2': int(pport[1]), 'Arg3': int(pport[2])})
```

Comment:
instead of vports[port] = (IxnetworkIxiaClientImpl.ixnet.Vport.add(Name=port), sport) (line 109)
we can do 
vports[port] = (IxnetworkIxiaClientImpl.ixnet..Vport.add(Name=port,Location=port.replace(":",";")), sport)

line 106,110,111 and 112 to 122 not required

instead of device type we can use port_type 
move AssignPorts after line 109 outside the for loop
and use ixn.AssignPorts(True)
and then get the port type if its vm or not
            portType = vports[port][0].Type